sections/semantics-embedded-content.include

```diff
@@ -2305,6 +2305,31 @@
   <a>boolean attribute</a>. The attribute must not be specified
   on an element that does not have an ancestor <{a}> element
   with an <{a/href}> attribute.
+  
+  <div class="example">
+    In this example the user is asked to identify a person in an image by
+    clicking on it. The server could respond to a click by confirming the
+    correct position or by asking the user to try again.
+  
+    <xmp highlight="html">
+      <figure>
+        <a href="solve.html">
+          <img src="zoo.jpg" ismap alt="Animals and people at the zoo, among them Wally with striped clothes">
+        </a>
+        <figcaption>Where’s Wally? Click on him!</figcaption>
+      </figure>
+    </xmp>
+  
+    The coordinates where the user clicks are send to the server with the
+    request for the resource referenced by the <{a}> element’s
+    <code>href</code> attribute.
+  </div>
+
+  <p class="note">
```

Comment:
This is helpful and I'll commit it this way for now, but I think we should actually move more of the description here. It would also be helpful to have a note explaining the accessibility issue with ismap - that if you don't have a pointer device, or can't see what it is pointing at, you're not "going to have a good time". Or to put it in spec language, you probably won't be able to use the map successfully.